docs/source/markdown/podman-systemd.unit.5.md

```diff
@@ -1914,6 +1922,23 @@ Override the default architecture variant of the container image.
 
 This is equivalent to the Podman `--variant` option.
 
+## Quadlet section [Quadlet]
+Some quadlet specific configuration is shared between different unit types. Those settings
+can be configured in the `[Quadlet]` section.
+
+Valid options for `[Quadlet]` are listed below:
+
+| **[Quadlet] options**      | **Description**                                   |
+|----------------------------|---------------------------------------------------|
+| DefaultDependencies=false  | Include implicit network dependencies to the unit |
+
+### `DefaultDependencies=`
+
+Add Quadlet's default dependencies to the unit.
+
+If set to true (the default), Quadlet will add a dependency (After=, Wants=) to `network-online.target` to the generated unit.
```

Comment:
Also here. Since `true` is the default, I think it's better to describe the outcome of setting it to `false`

Reply:
Updated for both!